sql/step_04_all_following_steps.sql

```diff
@@ -79,3 +79,7 @@ source step_82_anamnesis_patienthistory.sql;
 source step_83_fix_operationrow_column_names.sql;
 source step_84_session_table.sql;
 source step_85_remove_old_admission_fields.sql;
+source step_86_permissions_and_grouppermission.sql;
+source step_87_alter_table_opd.sql;
```

Comment:
yes i have update the  data of informatici in other to match the online database that intesys use for the test